lib/perl/Genome/InstrumentData/AlignmentResult/Merged.pm

```diff
@@ -153,8 +155,10 @@ sub create {
     my $self = $class->SUPER::create(@_);
     return unless ($self);
 
-    my $rv = eval {
+    $self->_lock_per_lane_alignment();
+    my @temp_allocations = ();
 
+    try {
         #TODO In a future version collect relevant alignments from other merged alignment results when available
```

Comment:
It looks like that `TODO` is from the original version of this module in 2011!  Now that you mention it, I believe the intention was to see if we could find an existing merged result and add our newly aligned lane to it instead of re-merging all the individual lanes again.  You're right that this isn't quite that... I wonder if that's still a thing we might want to do?